nuntium/templates/nuntium/profiles/contacts/contacts-per-writeitinstance.html

```diff
@@ -38,24 +38,24 @@
   <div class="page-header">
     <h2>{% trans "Recipients" %}</h2>
   </div>
-
+  <p>{% trans "There are" %} <strong>{{ writeitinstance.persons.count }}</strong> {% trans "recipients in your WriteIt" %}</p>
```

Comment:
@tmtmtmtm almost definitely, if a dev has two seconds I'm sure they could do this 1000% better